src/workload.rs

```diff
@@ -559,9 +563,11 @@ impl WorkloadStore {
                     .insert((service_sock_addr, port.target_port as u16));
             }
         }
-        if let Some(tx) = self.cert_tx.as_mut() {
-            if let Err(e) = tx.try_send(widentity) {
-                info!("couldn't prefetch: {:?}", e)
+        if Some(w.node) == self.local_node.clone() {
```

Comment:
why do we need to clone? if its needed can we do `Some(&w.node) == self.local_node.as_ref`

Reply:
if I just did `self.local_node` it would error out with `cannot move`. Changed it to that though.